crates/wast/src/wast.rs

```diff
@@ -54,9 +54,9 @@ impl<T> Outcome<T> {
     }
 }
 
-impl WastContext {
+impl<T> WastContext<T> {
```

Comment:
Oh how come this change was necessary? I didn't think we even gave access to the `Store`...

Reply:
This is used over in the fuzzing crate, which uses a resource limiter. So, while previously the store could have a `()` data and also own a `dyn ResourceLimiter`, now the store has to contain the `dyn ResourceLimiter`.